apps/spruce/README.md

```diff
@@ -221,3 +221,48 @@ following:
 ## Deployment
 
 Read more about deployment [here](../../packages/deploy-utils/README.md#deployment).
+
+
+## Advanced Debugging
+
+### Inspecting the State of the Application
+If debugging a feature requires you to inspect the application's state, tools like [React DevTools](https://react.dev/learn/react-developer-tools) can be very helpful. React DevTools is a browser extension available in Chrome, Firefox, and Edge that allows you to inspect the React component tree, view the state of components, and profile performance.
+
+#### Debugging in Safari
+Safari does not support React DevTools directly. To inspect the application state in Safari, you can follow these steps: (Steps 1-3 only need to be done your first time setting up React DevTools)
+
+1. **Install React DevTools CLI**:
+   ```bash
+   npm install -g react-devtools
+   ```
+
+2. **Create a Self-Signed Certificate**:
+   Generate a certificate to enable secure communication:
+   ```bash
+   openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout localhost.key -out localhost.crt -subj "/CN=localhost"
```

Comment:
does `-days` mean the certificate is valid for 365 days? So eventually after a year, you would have to repeat this step (contrary to the instructions at the top)?

Reply:
Yes I added a disclaimer